### PR TITLE
Populate form with json

### PR DESCRIPTION
**Pull request by dawnerd**

### CODE REVIEW

File: dist/formobj.js [hunk 1]
```diff
@@ -117,6 +117,92 @@ var FormObj = function () {
 
       return output;
     }
+  }, {
```
Comment (jgallen23) [inline dist/formobj.js]: should we remove dist from repo?

File: test/dist/test.js [hunk 2]
```diff
@@ -117,6 +117,92 @@ var FormObj = function () {
 
       return output;
     }
+  }, {
```
Comment (jgallen23) [inline test/dist/test.js]: can we remove this from repo?

**Comment by jgallen23** [comment]
we should add a prepublish step to package.json so we make sure that npm has the latest dist files

**Comment by jgallen23** [comment]
see https://github.com/firstandthird/scroll-triggers

**Comment by dawnerd** [comment]
There is one https://github.com/firstandthird/formobj/pull/1/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R16

File: package.json [hunk 3]
```diff
@@ -53,5 +54,10 @@
     "nswatch": "^0.2.0",
     "static-server": "^2.0.3",
     "tape": "^4.6.2"
+  },
+  "babel": {
```
Comment (jgallen23) [inline package.json]: this caused problems for me before

**Pull request merged**